assets/js/data.js

```diff
@@ -0,0 +1,158 @@
+    sourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby on rails', 'css', 'JavScript', 'html'],
+    featuredImage: '../assets/images/portfolio-3.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby on rails', 'css', 'JavScript', 'html'],
+    featuredImage: '../assets/images/portfolio-4.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby on rails', 'css', 'JavScript', 'html'],
+    featuredImage: '../assets/images/portfolio-1.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    tags: ['Ruby on rails', 'css', 'JavScript', 'html'],
+    featuredImage: '../assets/images/portfolio-2.png',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+
+];
+// Project card population
+function projectFunction() {
+  let projectCard = '';
+  // projectData.forEach(projectFunction);
```

Comment:
- [ ] Kindly remove commented unused pieces of code here and anywhere else as it is best practice and it would make your project less messy and more readable 👍